lib/foreman_ansible_core/ansible_runner_task_launcher.rb

```diff
@@ -23,18 +23,24 @@ def refresh
         if super
           @counter ||= 1
           @uuid ||= File.basename(Dir["#{@root}/artifacts/*"].first)
+          job_event_dir = File.join(@root, 'artifacts', @uuid, 'job_events')
           loop do
-            job_event_dir = File.join(@root, 'artifacts', @uuid, 'job_events')
-            event = Dir["#{job_event_dir}/#{@counter}-*"].first
-            return if event.nil?
-            @counter += 1 if handle_event_file(event)
+            files_with_nums = Dir["#{job_event_dir}/*.json"].map do |file|
```

Comment:
I couldn't think of more effective way that would be able to handle the case with missing number